src/assets/styles/popper.scss

```diff
@@ -0,0 +1,126 @@
+$pointer_arrow_size: 5px;
+
+@mixin popperPointer() {
+  :host {
+    margin: $pointer_arrow_size;
+  }
+
+  .container:after {
+    position: absolute;
+    content: " ";
```

Comment:
I'm not certain that this is intentional, but I think the idiomatic "empty" pseudo-element should have `content: ""` rather than `" "`.